CMakeLists.txt

```diff
@@ -317,6 +276,17 @@ endif()
 set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE CXX)
 set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD ${CMAKE_CXX_STANDARD})
 
+# We need to force skip code signing, or iOS will not comiple in CI due to XCode.
+# This only applies to iOS as MacOS doesn't care.
+if (IOS)
+    set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED "NO")
+    set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "")
+    set(XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "")
+    set(XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED "NO")
+    set_target_properties(${PROJECT_NAME} PROPERTIES
+        MACOSX_BUNDLE_INFO_PLIST ${CMAKE_CURRENT_SOURCE_DIR}/plist.in)
+endif()
```

Comment:
We only need to set `CMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED` and I have [set it as part of CI ](https://github.com/awslabs/aws-crt-cpp/pull/500/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR235)instead of in cmake so that it doesn’t affect other people building from source.

Reply:
Ah, I didn’t think about that. I don’t want to change it for everyone… I’ll adjust it! Thanks for catching that!